run_strategy_indexer.py

```diff
@@ -0,0 +1,49 @@
+"""
+A script to check that all strategy modules have been included in
+`./docs/reference/all_strategies.rst`
+"""
+import glob
+import sys
+import os
+
+def read_index(index_path):
+    """Read the index of strategies"""
+    with open(index_path, "r") as f:
+        strategies_reference = f.read()
+    return strategies_reference
+
+def get_module_name(module_path):
+    """
+    Take string of the form `./axelrod/strategies/titfortat.py` and returns
+    `titfortat`.
+    """
+    filename = os.path.basename(module_path)
+    module_name = os.path.splitext(filename)[0]
+    return module_name
+
+def check_module(module_path,
+                 index_path="./docs/reference/all_strategies.rst",
+                 excluded=("_strategies", "__init__", "_filters", "human")):
+    """
+    Check if a module is contained in the index of strategies
```

Comment:
Can we improve the docstring? Technically these aren't modules. Also let's use the standard style for parameters (just module_path is ok?) and the return value.